extensionsGallery-insider.json

```diff
@@ -2564,11 +2621,11 @@
 					"metadataItems": [
 						{
 							"name": "TotalCount",
-							"count": 42
+							"count": 44
```

Comment:
Was this not updated correctly last time?

Reply:
yes, most the time people don't update this.  We don't use it for anything, so perhaps we should remove it, but in the meantime I try to update it to the correct value whenever editing these files.